snorkel/labeling/model/logger.py

```diff
@@ -31,7 +31,7 @@ def check(self) -> bool:
             Whether to log or not based on logging frequency
         """
         self.unit_count += 1
-        return self.unit_count - 1 % self.log_train_every == 0
+        return (self.unit_count - 1) % self.log_train_every == 0
```

Comment:
can we do `return (self.unit_count) % self.log_train_every == 0` here and start with `send.unit_count = -1` in the `__init__` function? then it prints at epoch 0, 10, ... instead of 1, 11, ...